newIDE/app/src/MainFrame/EditorContainers/HomePage/CreateSection/index.js

```diff
@@ -551,7 +552,8 @@ const CreateSection = ({
                         onCreateProjectFromExample(
                           exampleShortHeader,
                           newProjectSetup,
-                          i18n
+                          i18n,
+                          true // isQuickCustomization
```

Comment:
Is the info in the exampleShortHeader already?

Reply:
No it's not unfortunately, the original example is used for the QC template